scripts/tile_benchmark/benchmark.py

```diff
@@ -0,0 +1,365 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+#############################################################################
+#  Copyright Kitware Inc.
+#
+#  Licensed under the Apache License, Version 2.0 ( the "License" );
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#############################################################################
+
+
+import math
+import sys
+
+import matplotlib.pyplot as plt
+import numpy as np
+import requests
+
+np.set_printoptions(threshold=np.nan)
+
+URL = 0
+NUM_LEVELS = 1
+NUM_X_TILES = 2
+NUM_Y_TILES = 3
+ATLAS_ARR = 4
+
+
+def main():
+    """Read in request data from source.txt, request tiles from each server and
+    visualize response times
+    """
+    file_name = 'source.txt'
```

Comment:
Great idea to put all these parameters in a config file. Can we make the path to the config file be the one command line argument to this script, instead of hardcoding it to `source.txt`?